### ISSUE TITLE
Mission:ISS Crashing on Launch

### ISSUE THREAD

**Issue by Jarmat** [LibreVR/Revive#571]
Haven't been able to get Mission:ISS to launch with Revive.  The app crashes immediately after launch with no errors.  Reinstalling didn't resolve the issue.

**Comment by Armada651** [comment]
I'm unable to reproduce this issue, it works fine for me. Can you check if there are any crashing log or dump files in the game directory?

**Comment by Jarmat** [comment]
Thanks for the quick response, I don't know why I didn't think to look for a log file... I found one in the MissionISS_Data directory.  In particular I found this:

ovr_GetAudioDeviceInGuid failed: The system or component has not been initialized.
Cannot get preferred VR audio IN device.Application Paused: HMD: False Focus: False HMD Present: False
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)

HMD Was Disconnected. Quitting Application.

Any ideas why I would get an HMD not found error?  I will point out that my other Oculus titles work with Revive with no issues.  Thanks! 

[output_log.txt](https://github.com/LibreVR/Revive/files/924350/output_log.txt)

**Comment by Armada651** [comment]
That's just a consequence of this problem:

```
ovr_Initialize(&initParams) failed: 
Compositor failed to initialize.ovr_Initialize(&initParams) failed: 
Compositor failed to initialize.UnloadTime: 0.899608 ms
```

Do you have SteamVR running when attempting to start Mission:ISS? Do you have the latest version of Revive (1.1) installed?

**Comment by Armada651** [comment]
From what I can see this may be an access problem. You've likely started SteamVR as an admin, but Revive was started as a normal user. This means Revive won't be able to access the compositor.

**Comment by Jarmat** [comment]
Yes, SteamVR was running at the time.  I'll have to check if it has the
latest version of Revive.

**Comment by Jarmat** [comment]
Ok thanks,  I'll make sure it runs as admin.

**Comment by Armada651** [comment]
Actually it would be better if you could run them both as normal user, but if you run them as admin at least you can confirm it's an access problem.
